components/top-nav/src/atoms/logo-anchor/index.js

```diff
@@ -0,0 +1,16 @@
+import styled from 'react-emotion';
```

Comment:
hmmm - i'm not entirely sure that this is an `atom` per-se, as instead it's a component generator - give it an anchor type and it'll return you a component...

Reply:
instead this is really a HOC, like `asTopNavAnchor` is